pyomo/contrib/cp/tests/test_logical_to_disjunctive.py

```diff
@@ -649,8 +817,36 @@ def test_transform_model(self):
         # and everything on the block is transformed too
         transBlock = m.block._logical_to_disjunctive
         self.assertEqual(len(transBlock.auxiliary_vars), 2)
-        self.assertEqual(len(transBlock.transformed_constraints), 7)
+        self.assertEqual(len(transBlock.transformed_constraints), 8)
         self.assertEqual(len(transBlock.auxiliary_disjuncts), 2)
         self.assertEqual(len(transBlock.auxiliary_disjunctions), 1)
         self.check_and_constraints(a, b1, transBlock.auxiliary_vars[1], transBlock)
         self.check_block_exactly(a, b1, b2, transBlock.auxiliary_vars[2], transBlock)
+
+    @unittest.skipUnless(gurobi_available, "Gurobi is not available")
+    def test_reverse_implication_for_land(self):
+        m = ConcreteModel()
+
+        m.t = BooleanVar()
+        m.a = BooleanVar()
+        m.d = BooleanVar()
+
+        m.c = LogicalConstraint(expr=m.t.equivalent_to(m.a.land(m.d)))
+
+        m.a.fix(True)
+        m.d.fix(True)
+
+        m.binary = Var(domain=Binary)
+        m.t.associate_binary_var(m.binary)
+
+        m.obj = Objective(expr=m.binary)
+
+        TransformationFactory('contrib.logical_to_disjunctive').apply_to(m)
+        TransformationFactory('gdp.bigm').apply_to(m)
+
+        # Should be 1 (we forced it), but we get 0!
```

Comment:
Is this comment still accurate or just a note from debugging?

Reply:
Ah, good catch, after the comma is a lie.